README.md

```diff
@@ -65,9 +65,9 @@ gcloud dataproc clusters create "$MY_CLUSTER"
 ## Downloading and Using the Connector
 
 The latest version connector of the connector is publicly available in 
-[https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-latest.jar](https://console.cloud.google.com/storage/browser/spark-lib/bigquery).
+[gs://spark-lib/bigquery/spark-bigquery-latest.jar](https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-latest.jar).
 A Scala 2.12 compiled version exist in
-https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-latest_2.12.jar.
+[gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar](https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-latest.jar).
```

Comment:
Please change the link to https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-latest_2.12.jar

Reply:
Done.